### PR TITLE
Moving the 'testing your web app' page to the GSG

### PR DESCRIPTION
**Pull request by davidrichards-da**
CHANGELOG_BEGIN
CHANGELOG_END

Moving the 'testing your web app' page to the GSG

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [x] Normal production system change, include purpose of change in description
- [ ] If you mean to change the status of a component, please make sure you keep [the Component Status page](https://github.com/digital-asset/daml/blob/main/docs/source/support/component-statuses.rst) up to date.

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -21,6 +21,7 @@ Daml Documentation
    Building Your App <getting-started/index>
    getting-started/app-architecture
    getting-started/first-feature
+   getting-started/testing
```
Comment (cocreature) [inline docs/source/index.rst:24]: you also need to update the pdf version of this in `docs/configs/pdf/index.rst`.

Comment (cocreature) [reply docs/source/index.rst:24]: and mirror those changes in the assembly repo.

Comment (stefanobaghino-da) [reply docs/source/index.rst:24]: I had a look at what's in the assembly repo. It appears there's a shorter version of the table of contents. What needs to be replicated? Where is this process documented? Who owns it?

Comment (cocreature) [reply docs/source/index.rst:24]: In https://github.com/DACH-NY/assembly/blob/main/docs/index/index_html.rst we need to add it to the getting started guide, similar for https://github.com/DACH-NY/assembly/blob/main/docs/index/index_pdf.rst.

Do you have an idea of where this should be documented such that people actually read it? Maybe just as a comment in this file? In the readme in the assembly repo? Somewhere else?

Comment (stefanobaghino-da) [reply docs/source/index.rst:24]: Could `docs/README.md` be a good place for these pointers to be?

Btw my question was not about where we need to fill the table of contents, but what needs to be there. It looks like a shorter version of the original table of contents but I cannot tell what the rationale is to include or not include something.

Comment (cocreature) [reply docs/source/index.rst:24]: In the end, the one in this repo is only to get something for easy testing. The authorative ToC is only in the assembly repo the one here is not used at all in the final build. The one in the assembly repo is just structured differently, it still includes everything but sometimes at a deeper level (if something is unreachable from the toc sphinx throws an error).

i’ll try to add something to the readme later today.

Comment (stefanobaghino-da) [reply docs/source/index.rst:24]: Thanks. How can I test whether the documentation in the assembly repo produces the artifact I would expect?

Comment (cocreature) [reply docs/source/index.rst:24]: https://github.com/DACH-NY/assembly/blob/main/docs/README.md but that currently only describes how to deal it with a released version. It is possible to test against a branch but that is currently undocumented and very hacky.

**Review by cocreature — COMMENTED** [review]
CI is failing because you have some broken references in `first-feature.rst`.

**Review by cocreature — APPROVED** [review]
thanks!

**Comment by cocreature** [comment]
you still have more broken links `/tmp/tmp.66dFhtxkvo/source/source/daml/intro/12_Testing.rst:13:unknown document: /app-dev/bindings-ts/testing`. I recommend to test locally before pushing.

**Pull request merged**